### DISCUSSION TITLE
Modify the storage path

### DISCUSSION THREAD

**Discussion by 1372764481** [ultralytics/yolov5#8066] [Q&A]
I don't want to create a new exp file after each identification, but I want to store the results of each identification in the first exp directory.

**Comment by glenn-jocher** [comment]
@1372764481 see argparser
https://github.com/ultralytics/yolov5/blob/a80dd66efe0bc7fe3772f259260d5b7278aab42f/detect.py#L231-L233

**Reply by 1372764481** [reply]
Thank you for your answer, but I'd like to know which function uses these parameters. I've never found it.

**Reply by glenn-jocher** [reply]
@1372764481 all functions that log results use these arguments.
